src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioner.scala

```diff
@@ -56,6 +62,13 @@ trait LociPartitionerArgs
     handler = classOf[BooleanOptionHandler]
   )
   var quiet: Boolean = false
+
+  @Args4JOption(
+    name = "--allow-ungrouped-contigs",
+    usage = "Loosen requirement that regions are grouped by contigs.",
```

Comment:
yea, I thought it was simply less efficient but still correct on unsorted data, but digging back in, that is not true, so i've added verifications to the `ContigsIterator` and `ContigCoverageIterator` making sure these assumptions hold. thanks!